executor/aggregate.go

```diff
@@ -187,6 +187,7 @@ type HashAggExec struct {
 	// After we support parallel execution for aggregation functions with distinct,
 	// we can remove this attribute.
 	isUnparallelExec bool
+	initialized      bool
```

Comment:
This field seems to only meaningful to the "parallelExec"... Should we rename it to `parallelExecInitialized`?